lms/static/js/spec/student_profile/badge_list_container_spec.js

```diff
@@ -0,0 +1,85 @@
+define(['backbone', 'jquery', 'underscore', 'URI', 'common/js/spec_helpers/ajax_helpers',
+        'js/spec/student_profile/helpers',
+        'js/student_profile/views/badge_list_container',
+        'common/js/components/collections/paging_collection'
+    ],
+    function (Backbone, $, _, URI, AjaxHelpers, LearnerProfileHelpers, BadgeListContainer, PagingCollection) {
+        "use strict";
+        describe("edx.user.BadgeListContainer", function () {
+
+            var view, requests;
+
+            var createView = function (requests, badge_list_object) {
+                var badgeCollection = new PagingCollection();
+                badgeCollection.url = "/api/badges/v1/assertions/user/staff/";
+                var models = [];
+                _.each(_.range(badge_list_object.count), function (idx) {
+                    models.push(LearnerProfileHelpers.makeBadge(idx));
+                });
+                badge_list_object.results = models;
+                badgeCollection.fetch();
+                var request = AjaxHelpers.currentRequest(requests);
+                var path = new URI(request.url).path();
+                expect(path).toBe("/api/badges/v1/assertions/user/staff/");
+                AjaxHelpers.respondWithJson(requests, badge_list_object);
+                var badge_list_container = new BadgeListContainer({
+                    'collection': badgeCollection
+
+                });
+                badge_list_container.render();
+                return badge_list_container;
+            };
+
+            afterEach(function () {
+                view.$el.remove();
+            });
+
+            it("All badges are displayed", function () {
```

Comment:
Nit: the best practice for test names is that they should read like a sentence starting with "it". Currently this reads as "it all badges are displayed". I'd make it something more like:

    it("displays all badges", function() {